### PR TITLE
es_DSDEEPB-1500 for putting attributes in the summary area

### PR DESCRIPTION
**Pull request by eddiebroad**
more progress

some progress

got saving to work

some tidy

got delete working

fixed an unused item which caused a compilation issue

some tidy

arranged the attributes floating to the right

tidy

fixed an improper arity

set focus to last of fields

remove TODO message

fix bug

got focus working

### CODE REVIEW

**Comment by MattPutnam** [comment]
A few issues:
* This is not filtering out the "description" property (if present)
* The layout is poor.  The location of the attributes jumps when you go to edit them.  My suggestion would be to move the "Created By" and "Submissions" sections to the right column (and move that column over further), and put the attributes below the bucket.  Maybe the bucket could be on the right too.
* There is no way to cancel editing, aside from bailing on the page.
* The Clone, Lock, and Delete buttons should be hidden when editing.

**Comment by eddiebroad** [comment]
Re : "This is not filtering out the "description" property (if present)"  What is meant by this? Is it mean a workspace description?  Can clarification be provided?

**Comment by MattPutnam** [comment]
We hijack a "description" attribute to use for the workspace description, since there isn't a real field for it.  So we need to filter out that attribute.

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs [hunk 1]
```diff
@@ -114,13 +144,8 @@
                (let [owner? (= "OWNER" (workspace "accessLevel"))
                      writer? (or (= "WRITER" (workspace "accessLevel")) owner?)
                      status (common/compute-status workspace)]
-                 (if (:viewing-attributes? @state)
-                   [AttributeViewer {:ws workspace :writer? writer? :on-done #(swap! state dissoc :viewing-attributes?)
-                                     :attrs-list (mapv (fn [[k v]] [k v])
-                                                   (dissoc (get-in workspace ["workspace" "attributes"]) "description"))
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs]: Here's where the old version was filtering out "description"

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs [hunk 2]
```diff
@@ -56,98 +56,66 @@
...
+          (:attrs-list @state))
+        [:div {:style {:display (when-not (:editing? @state) "none")}}
+         [comps/Button {:style :add :text "Add new"
+                        :onClick
+                        #(do
+                          (swap! state update-in [:attrs-list] conj ["" ""])
+                          (js/setTimeout (fn [] (common/focus-and-select
+                                           (-> (@refs (str "field" (- (count (:attrs-list @state)) 1))) .getDOMNode))) 250 )
+                          )
+                        }]]])
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs]: Fix the formatting around here, and I think the timeout delay can be quite a bit shorter.

**Pull request closed without merging**